### PR TITLE
O2 new brand

### PR DESCRIPTION
**Pull request by yceballost**

### CODE REVIEW

**Comment by aweell** [comment]
There are some cases that we need to check:

* Snap card inverse and snapCard inverse over inverse
* boxedRow inverse and boxedRow inverse over inverse
* accordion boxed inverse and accordion boxed overInverse

[Example](https://mistica-aig5bz292-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AlGBnADhAduglgG4wAyAhgJ4QCuALgAT7oCSOxATujAHwA6OSAEIQAHjCgBBMJHZR8uenxz16Q0eKky5uZrRgBbRizYxOMfipVl0XWgF5gSZpBytCEfGBgAFADZkcADF8Xxg0AHNqf3Z6AHpuAF8LS1p8WlC7XhAAdQALMgYmegBZCEImWjIY0pwYCizklRc9HHtgRsskABUYEVoAZnpIXwh2B2HR9AA6PT6AZRgXKCqKBKVLDfpdAHJ0ejJ6EYD9wjIQsgAjUPpaCH2cKlqAGnoAd1yYWg%2BY0focCAYVGo23YMH2JTKFSqHQ2YGo6Fu%2BlMUy2DAgADN0aY9uj2BBDIAaggATE8AAzkm53AkARgArGSKa80rlwQR9JhrgEoPR0dZaDDLGRMBzPAV5MpMHivDYXlz6ED6KCvEQwV8wfpcHVGMpQjYbvllAAWAAc9FyNE4UwFSFiPT6-XWliSyjijrUYkk0lG2lcen0HWstgcThcJAgYDFuGKQoiUSqrudG1S6RgmRAXTu73xRkYDDOSKgAH4Gi6mrgWm1rXaBkMICMxsAJpbZrQFksVmsBSouh8IeUEfGalqjsouOxyl4c-nxPs8w9NS93p9vvKaPtQft2iB6FlE5sVN1erQSPgcABrM2g9HcUr9yrsG3Vk-n7hDeGI0z0X7-WhW0udW0jwdDo93iZJ3Q0L1ZHFXQDADGxPmDZxcDDMAzwAUQoGAAGERi4KBY2iBMOmTDIsjyO4MSxTg%2ByhapNQoYsQA6ZoYFaBwqyA2t63GOtJhmI821wZZ2FWR19yQTBxP3egJAYHpQnRABnnBRUXMF8mIL9qB%2BV5lHRU8AjAfAyF8bUETSOhxReBSYHRXBRS7SxsNwdBqCRGJghwIyYFlHBuTIcJQQMNjaD2Jkvi-NUYlPCzaCs1zKX2GwKiclQFVPL8LkqU9T3CVcdMOCAAj-GTVFiKSnKQC4YjA-8NicfRArBdB2DANNcloWhMHQeBYgqzw3P0KZMHNW50FiakiXJSbptJLJkswRZaBQSM02pAA2eAAE4Frqsrqtq6SGsq%2BrNgAERgJb-Lyr9lDVIZ8nYMgwD0dgKkGr90X1MFT3s9gmtSBQFUlSFYHoeEnjSgqYiFEUIyB5QmV8MyLha0Kks1L6fvM5MErwLcdxAPcDqfU8L1yK9uAill3k8Flbzos1rH2IKYBC1p0EfI9nzPbhSpkm1TokwD7Wk0C3WED1NG9GC-XgoNHGQ1woGwqoCJgSIiPiEmbjSMiQAACQgV4hmOZgzXwDc0iYljy1CjizsPe1uNGXj62mFshP8jtjssHY9gODgKFZkIwVB6U9i%2BAoCrN5QXsj-U8WocIWTVJyNVqCgpg1O8qimDA3hgC4CD0FEAE01wjfTT25DUN1gHLQm5P7RkB8UnMuGgGAeiOMD2BQLSJ43Td0ehslGM90AWzBmoFhrRYGcXkn21QpcgrQ5bg-9A0QpXQ3wLFInVwIjhWzW4yO3XSNTcj8gYdBzSibkLagO5D62eg35wbYGCVGAVQ4zmMUOYbxmQ42aM9V6t1ICwFtv%2BVi7F2hO2rIMJs7t%2BJe0WMJX2Tkx5NRDpUM8YIDhYlNvoU8dBC7UFaCEWO-9AEPWAaAmmEDoAwBRMwdETkFQWRRszYgP8-6LAAcQbkD1YG%2BVjtXRUMAACO1AMB5j%2BDAU2WMLj1DOk0Xwngzy3QUFkUEXB-L0GYVaZiTtF7AX-BLfgNp16ek3rgJQNo0BYFciqcgQJaDcBAE8EAaokToAQAAbRAAAeSJAAfQAHKqJAAAXX8UyKAXxgnwBCf0DapIEkJCAA)

File: tokens/o2-new.json [hunk 1]
```diff
@@ -0,0 +1,1646 @@
...
+          {
+            "value": "rgba({palette.grey60}, 0)",
+            "stop": 0
+          },
+          {
+            "value": "rgba({palette.grey60}, 0.4)",
+            "stop": 0.3
+          },
+          {
+            "value": "rgba({palette.grey60}, 1)",
```
Comment (aweell) [inline tokens/o2-new.json]: Should in dark the opacity of the grey60 be 0.7?

File: tokens/o2-new.json [hunk 2]
```diff
@@ -0,0 +1,1646 @@
...
+      "value": "{palette.beyondBlue40}",
+      "type": "color",
+      "description": "beyondBlue40"
+    },
+    "textLinkDanger": {
+      "value": "{palette.o2Red45}",
+      "type": "color",
+      "description": "o2Red45"
+    },
+    "textLinkSnackbar": {
```
Comment (HectorSancho) [inline tokens/o2-new.json:963]: beyondblue30

File: tokens/o2-new.json [hunk 3]
```diff
@@ -0,0 +1,1646 @@
...
+      "type": "color",
+      "description": "grey45"
+    },
+    "controlActivated": {
+      "value": "{palette.beyondBlue30}",
+      "type": "color",
+      "description": "beyondBlue30"
+    },
+    "controlInverse": {
+      "value": "{palette.darkModeGrey6}",
```
Comment (HectorSancho) [inline tokens/o2-new.json]: grey45

File: tokens/o2-new.json [hunk 4]
```diff
@@ -0,0 +1,1646 @@
...
+      "type": "color",
+      "description": "darkModeGrey6"
+    },
+    "controlActivatedInverse": {
+      "value": "{palette.beyondBlue30}",
+      "type": "color",
+      "description": "beyondBlue30"
+    },
+    "controlError": {
+      "value": "{palette.o2Red60}",
```
Comment (HectorSancho) [inline tokens/o2-new.json]: o2Red45

**Review by aweell — APPROVED** [review]

File: tokens/o2-new.json [hunk 5]
```diff
@@ -0,0 +1,1646 @@
...
+      "type": "color",
+      "description": "grey60"
+    },
+    "neutralMediumInverse": {
+      "value": "{palette.grey60}",
+      "type": "color",
+      "description": "grey60"
+    },
+    "neutralLow": {
+      "value": "{palette.darkModeGrey6}",
```
Comment (HectorSancho) [inline tokens/o2-new.json]: cambiar grey80

File: tokens/o2-new.json [hunk 6]
```diff
@@ -0,0 +1,1646 @@
...
+      "type": "color",
+      "description": "darkModeGrey6"
+    },
+    "toggleAndroidInactive": {
+      "value": "{palette.grey20}",
+      "type": "color",
+      "description": "grey20"
+    },
+    "toggleAndroidBackgroundActive": {
+      "value": "{palette.o2BlueLight30}",
```
Comment (HectorSancho) [inline tokens/o2-new.json]: beyondBlue20

**Review by HectorSancho — CHANGES_REQUESTED** [review]

File: tokens/o2-new.json (hunk 6 above)
Comment (yceballost) [reply tokens/o2-new.json]: beyondBlue15

**Pull request merged**